src/controllers/message.controller.js

```diff
@@ -0,0 +1,36 @@
+import { Message } from '../models/Message.model.js';
+import { EventEmitter } from 'events';
+
+const MESSAGE = 'message';
+
+export const emitter = new EventEmitter();
+
+export const getMessages = async (req, res) => {
+  const whereClauses = {};
+  const { roomId } = req.query;
+
+  whereClauses.roomId = roomId;
+  res.send(await Message.findAll({ where: whereClauses }));
+};
+
+export const postMessage = async (req, res) => {
+  const { text, author, roomId } = req.body;
+
+  if (!text || !author || !roomId) {
+    res.status(400).send({ message: 'one of nessesery parys is required' });
+  }
+
+  const message = {
+    text,
+    time: new Date().toString(),
+    author,
+    roomId,
+  };
+
+  if (!message) {
+    res.status(400).send({ message: 'message is required' });
+  }
```

Comment:
The condition in line 30 is unnecessary. The object `message` is created just before and it will always exist, even if some properties are undefined. Therefore, this check will never result in a 400 status response. It's better to check each property individually before creating the `message` object.